libs/cart/testing/src/factories/cart-item/cart-item.factory.spec.ts

```diff
@@ -39,6 +39,11 @@ describe('Cart | Testing | Factories | CartItemFactory', () => {
       expect(result.price).not.toBeNull();
       expect(result.row_total).not.toBeNull();
       expect(result.in_stock).not.toBeNull();
+      expect(result.discounts).not.toBeNull();
+    });
+
+    it('should set the sum of discounts to be less or equal to than price', () => {
```

Comment:
Looks like there's a typo somewhere in here

Reply:
fixed